src/weights.jl

```diff
@@ -281,6 +288,9 @@ Base.mean{T<:Number,W<:Real}(A::AbstractArray{T}, w::WeightVec{W}, dim::Int) =
 
 
 ###### Weighted median #####
+function Base.median(v::AbstractArray, w::WeightVec)
```

Comment:
Shouldn't this method be unnecessary since all you're doing is throwing a `MethodError`? Presumably without the method, it would also be a `MethodError`.